synapse/lib/stormtypes.py

```diff
@@ -5194,6 +5194,18 @@ class User(Prim):
                       {'name': 'iden', 'type': 'str', 'desc': 'The iden of the Role.', },
                   ),
                   'returns': {'type': 'null', }}},
+        {'name': 'grants', 'desc': '''
+        Grant a list of Roles to the User.
```

Comment:
Feels like maybe this should use the 'Replace all the roles for a given user with a new list of roles.' description from `HiveAuth.grants` just in case they don't read the note that it replaces the existing roles.

Reply:
I could see that being confusing. I'm totally open to other names which are clearer. ``grants`` just came to mind when I was thinking of the "smash in a list of roles".

Reply:
how about `setRoles()` to mirror the (requested) API name?